.github/workflows/rust.yml

```diff
@@ -58,9 +67,10 @@ jobs:
       - uses: actions-rs/cargo@v1
         env:
           RUSTFLAGS: '--cfg debug_assertions'
+          RUSTC_FLAGS: ${{ matrix.rustc-args }}
```

Comment:
The name is wrong. Those flags aren't passed to rustc. They are passed to the test binary.

Reply:
is `TEST_FLAGS` better?

Reply:
Yes! But you also need to rename `rust-args` to `test-args`. :D